include/openssl/aead.h

```diff
@@ -244,6 +244,10 @@ struct evp_aead_ctx_st {
 // be used.
 #define EVP_AEAD_DEFAULT_TAG_LENGTH 0
 
+// EVP_AEAD_AES_256_GCM_IV_LEN is the number of bytes required for 96 bits
+// of IV to be used in AES-GCM.
+#define EVP_AEAD_AES_256_GCM_IV_LEN 12
```

Comment:
the `AES_GCM_NONCE_LENGTH` in [`crypto/fipsmodule/cipher/e_aes.c`](https://github.com/awslabs/aws-lc/blob/08b279d654fdb393c33de529e2b6244f9d6c8fdd/crypto/fipsmodule/cipher/e_aes.c) seems to imply the same value. Is it possible to reuse this?

Reply:
Good call! I'd been looking for a useful `#define`, but only in the `.h` files.